source/mir/random/generic/package.d

```diff
@@ -0,0 +1,261 @@
+/**
```

Comment:
Should be `mir/random/tinflex` instead of `generic` because other generic approaches are possible